TCD1304.py

```diff
@@ -77,11 +73,9 @@ def analog_signal_read (self, block:False):
         self.scope.select_range('CH1', max_voltage_output)   # voltagerange should be fitted to the sensors output for better resolution. sensor otput is between 2V-3V due to datasheet
         self.scope.configure_trigger(channel = 'CH1', voltage = min_voltage_output ) # starts recording , when voltage is over a certain level
         self.scope.capture(channels=1, samples=integration_elements*samples_per_element, timegap=integration_time/samples_per_element, block=False)
-        self.icg_clock(self)
-        x, y= self.scope.fetch_data()
-        diff = abs(x,y[1, 0] - min_voltage_output )
-        analog_measurement = np.ndarray ()
-        return analog_measurement  # this should put the Measurements and timestamps in an array
+        icg_clock()
+        x, = self.scope.capture()  # putting timestamps into numpy array (don't know, if that is necessary for our purpose)

```

Comment:
You already call `capture` above, no need to call it again.

Reply:
Means I get rid of line 77 ...

Reply:
Yes.